packages/databricks-vscode/src/test/e2e/resources/dummy-test/package.json

```diff
@@ -4,25 +4,20 @@
     "description": "A dummy extension to run integration tests",
     "version": "0.0.1",
     "engines": {
-        "vscode": "^1.74.0"
+        "vscode": "^1.69.0"
```

Comment:
Did this break the tests?

Reply:
Yep. We need the dummy extension to support the same versions as the main extension (and hence wdio).